charts/victoria-logs-cluster/values.yaml

```diff
@@ -0,0 +1,1071 @@
+    useServiceMonitor: false
+    # -- Target namespace of VMServiceScrape manifest
+    namespace: ""
+    # -- VMServiceScrape labels
+    extraLabels: {}
+    # -- VMServiceScrape annotations
+    annotations: {}
+    spec:
+      endpoints:
+        - port: http
+          # -- Basic auth params for VMServiceScrape
+          # basicAuth: {}
+          # interval: 15s
+          # scrapeTimeout: 5s
+          # -- vmServiceScrape relabelings
+          # relabelings: []
+          # -- vmServiceScrape metricRelabelings
+          # metricRelabelings: []
+
+vlinsert:
+  # -- Enable deployment of vlinsert component.
+  enabled: true
+  # -- Override default `app` label name
+  name: ""
+  # -- vlinsert strategy
+  strategy: {}
+   # rollingUpdate:
+   #  maxSurge: 25%
+   #  maxUnavailable: 25%
+   # type: RollingUpdate
+  image:
+    # -- Image registry
+    registry: ""
+    # -- Image repository
+    repository: victoriametrics/victoria-logs
+    # -- Image tag
+    # override Chart.AppVersion    
+    tag: ""
+    # -- Image tag suffix, which is appended to `Chart.AppVersion` if no `server.image.tag` is defined
+    variant: victorialogs
+    # -- Image pull policy
+    pullPolicy: IfNotPresent
+  # -- Specify pod lifecycle
+  lifecycle: {}
+  ports:
+    # -- vlinsert http port name
+    name: "http"
+  # -- Name of Priority Class
+  priorityClassName: ""
+  # -- Overrides the full name of vlinsert component
+  fullnameOverride: ""
+  # -- Extra command line arguments for vlinsert component
+  extraArgs:
+    envflag.enable: true
+    loggerFormat: json
+    httpListenAddr: :9481
+  # -- Deployment annotations
+  annotations: {}
+  # -- Deployment additional labels
+  extraLabels: {}
+  # -- Pod’s additional labels
+  podLabels: {}
+  terminationGracePeriodSeconds: 30
+
+  # -- Additional environment variables (ex.: secret tokens, flags). Check [here](https://docs.victoriametrics.com/#environment-variables) for details.
+  env: []
+
+  # -- Specify alternative source for env variables
+  envFrom: []
+    #- configMapRef:
+    #    name: special-config
+
+  # -- Suppress rendering `--storageNode` FQDNs based on `vlstorage.replicaCount` value. If true suppress rendering `--storageNode`, they can be re-defined in extraArgs
+  suppressStorageFQDNsRender: false
+
+  # -- IDs of vlstorage nodes to exclude from writing
+  excludeStorageIDs: []
+
+  # -- Readiness & Liveness probes
+  probe:
+    # -- vlinsert readiness probe
+    readiness:
+      httpGet: {}
+      initialDelaySeconds: 5
+      periodSeconds: 5
+      timeoutSeconds: 5
+      failureThreshold: 10
+    # -- vlinsert liveness probe
+    liveness:
+      tcpSocket: {}
+      initialDelaySeconds: 5
+      periodSeconds: 15
+      timeoutSeconds: 5
+      failureThreshold: 3
+    # -- vlinsert startup probe
+    startup: {}
+
+# Horizontal Pod Autoscaling
+  horizontalPodAutoscaler:
+    # -- Use HPA for vlinsert component
+    enabled: true
```

Comment:
hey @Haleygo, quick followup: is this intentionally true? it's the only component that has the hpa enabled by default

Reply:
Thank you for notice! 
Must be fixed in https://github.com/VictoriaMetrics/helm-charts/pull/2165.